src/styles/index.css

```diff
@@ -1,5 +1,17 @@
 /* NOTE: Use plain css for now. Switch to using Sass once someone get's it to load in storybook preview iframe. */
 
+/**
+ * Normalize
+ */
+html {
+  box-sizing: border-box;
+}
+*,
```

Comment:
This file is just for storybook, right?

I think we shouldn't apply exotic global rules like this even in storybook, since then our components will be implemented assuming that every site has these rules, and in reality applying a specific `box-sizing` to `*` is certainly not that common.

Most sites prefer the `normalize.css` way of applying `box-sizing` only to specific known elements, since applying it to `*` is a source of many a specificity issue.

Reply:
Actually this index.css is meant to be included to webpages eventually. Yeah, I thought about this and went for the generic normalize all box-sizing since otherwise we would need to specify it for all components separately. But yeah, I see your concern. Not sure what's the common want in WP for this?

Reply:
Ah, thanks for the clarification. I feel very strongly that a **component library** must under no circumstances provide global styles that break the rest of the site.

Reply:
@henritukiainen I think you are right. I'll remove the global box-sizing rules. Thanks for bringing this up!